sql/moz-fx-data-shared-prod/mozilla_org_derived/www_site_landing_page_metrics_v1/query.sql

```diff
@@ -0,0 +1,180 @@
+WITH sessions_table AS (
+  -- Pull sessions related activity by landing page
+  SELECT
+    date,
```

Comment:
```suggestion
    `date`,
```

nit: `date` is a keyword so you could wrap inside ` ` `  to make sure it is clear that this is a field name. For example, this is done with `language` below inside the same SELECT statement.